### ISSUE TITLE
How to get the exact date and time from ecliptic longitude on gps coordinates

### ISSUE THREAD

**Issue by ratobiajin** [mourner/suncalc#155]
Hello! I am thankful for the effort you have done for this project. In Chinese and Japanese culture there are 24 Solar Terms, the dates of which are calculated taking into account the ecliptic longitude by two methods — píngqì (平気) based on the mean motion of the Sun dividing the tropical year evenly into 24 periods, and dìngqì (定気) based on the position of the Sun on the ecliptic.  I'd like to have a script that calculates the exact local time (MM/DD hh;mm) for the given year at the location given by lat/lon GPS coordinates of main 8 Solar Terms, each of which take place at given ecliptic longitude, in both methods:

315° | 1 Lichun
0° | 2 Chunfen (Spring Equinox)
45° | 3 Lixia
90° | 4 Xiazhi (Summer Solstice)
135° | 5 Liqiu
180° | 6 Qiufen (Autumn Equinox)
225° | 7 Lidong
270° | 8 Dongzhi (Winter Solstice)

getSolarTerm(2022,4,"pingqi"), i.e. Xiazhi (Summer Solstice) of 2022, in case of Tokyo, Japan should return "06/21 18:14"
getSolarTerm(2022,4,"dingqi"), i.e. Xiazhi (Summer Solstice) of 2022, in case of Tokyo, Japan should return "06/22 04:55"

**Comment by camiteca** [comment]
Does #156 cover this case?

**Comment by ratobiajin** [comment]
> Does #156 cover this case?

No, unfortunately, it does not. I'd like to get date AND time from the provided GPS lat/lon coordinates AND ecliptic longitude.

**Comment by mourner** [comment]
Closing as out of scope for SunCalc — it's a different problem domain (calendar/ecliptic-longitude solving rather than rise/set/position) and, crucially, doesn't use lat/lon at all.
